From 7b23a14f9dff8bfbf9f84804c2e8fb87b6faca2d Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 7 Jan 2018 00:46:51 +0100
Subject: [PATCH 1/1] board_r: remove superfluous #ifdef CONFIG_PRAM

initr_mem() is already enclosed by
	#if defined(CONFIG_PRAM)
	#endif

So there is no need to check CONFIG_PRAM again inside the
function.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 common/board_r.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/common/board_r.c b/common/board_r.c
index 5e7eb2d5cb..98811edc07 100644
--- a/common/board_r.c
+++ b/common/board_r.c
@@ -624,9 +624,7 @@ int initr_mem(void)
 	ulong pram = 0;
 	char memsz[32];
 
-# ifdef CONFIG_PRAM
 	pram = env_get_ulong("pram", 10, CONFIG_PRAM);
-# endif
 	sprintf(memsz, "%ldk", (long int) ((gd->ram_size / 1024) - pram));
 	env_set("mem", memsz);
 
-- 
2.14.2

